components/brave_rewards/resources/page/components/adsBox.tsx

```diff
@@ -66,6 +66,82 @@ class AdsBox extends React.Component<Props, State> {
     )
   }
 
+  getAdsSubdivisions = () => {
```

Comment:
Anyway we can merge this the `getAdsSubdivisions()` method in `components/brave_rewards/resources/android_page/components/adsBox.tsx`?

Reply:
I have just seen this, and in a future update we will move the list from adsBox to the ads lib. Thanks